EmployeesSalaries/EmployeesSalaries/Models/Employee/HR.cs

```diff
@@ -7,11 +7,23 @@ public class HR : IEmployee, IReportsTo
         public string FirstName { get; set; } = "Test";
         public string LastName { get; set; } = "Test";
         public string Role { get; set; } = "HR";
-        public IEmployee Manger { get;  } = new Manager();
+        public IEmployee Manager { get; set; } = new Manager();
 
         public int Id { get; } = 3;
 
         public bool IsMatch(int id) { return id == Id; }
+        public bool ReportsTo(int id) { return id == Id; }
+        public IEmployee Assign(IEmployee manager) {
+            if (manager.Role == "Manager") 
```

Comment:
this is HR employee, and you're checking for manager role
it will not work correctly
please make sure to test your code before creating a PR